### PR TITLE
Restrict moodledata directory permissions

### PR DESCRIPTION
**Pull request by mroote**

### CODE REVIEW

File: systemd/postinst.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ echo "Checking Moodle data directory"
 if [ ! -d /opt/moodle/moodledata ]; then
     echo "Creating Moodle data directory"
     mkdir -p /opt/moodle/moodledata
-    chmod 777 /opt/moodle/moodledata
```
Comment (hrpatel) [inline systemd/postinst.sh:30]: Why not fix ownership and permissions in one script?

Comment (hrpatel) [reply systemd/postinst.sh:30]: Probably best to do it in the container at start

Comment (mroote) [reply systemd/postinst.sh:30]: You have to make the directory from the host, but set the permissions from within the container in order to have the correct owner on the directory.

Comment (hrpatel) [reply systemd/postinst.sh:30]: Good point... do the permissions on the host match the permission inside the container? If not, we can set it in `configure-moodle.sh`

**Review by hrpatel — APPROVED** [review]

**Pull request merged**